### DISCUSSION TITLE
Displaying several notifications simultaneously

### DISCUSSION THREAD

**Discussion by HexagonArchive** [linuxmint/discussions#274] [Ideas]
Picture this - you are receiving a number of very short messages and are forced to read them one by one with a period of a few seconds. This might become infuriating. It would be much better to show a couple notifications on screen at the same time as they appear, at least with a limit of a certain amount of messages. I suggest adding this feature and the ability to set the maximum amount of notifications shown on the screen simultaneously. It already exists on other desktop environments, such as MATE or GNOME on Ubuntu.

**Comment by rcalixte** [comment]
I think this is a duplicate of #217.

**Reply by igorsantos07** [reply]
Definitely. @HexagonArchive, any chance you could close this in favor of earlier one? It would help out with searches.
